File: 0001 Code Chef/0015 Election Hopes/implementation/0009 dart/election_hopes.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  if(stringInput != null){
    List<String> stringArray = stringInput.split(" ");
    int? x = int.tryParse(stringArray[0]!);
    int? y = int.tryParse(stringArray[1]!);
    if(x != null && y != null){
      if(x >= 2*y){
        print("YES");
      }else{
        print("NO");
      }
    }
  }
}

File: 0001 Code Chef/0015 Election Hopes/implementation/0010 nodejs/election_hopes.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});

rl.question('',(input) =>{
    string_array = input.split(" ");
    let x = parseInt(string_array[0]);
    let y = parseInt(string_array[1]);
    if(x >= 2*y){
        console.log("YES");
    }else{
        console.log("NO");
    }
    rl.close();
})